internal/pb/export/export.pb.go

```diff
@@ -21,12 +21,11 @@
 package export
 
 import (
-	reflect "reflect"
-	sync "sync"
-
 	proto "github.com/golang/protobuf/proto"
 	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
 	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
+	reflect "reflect"
+	sync "sync"
```

Comment:
This returns the protofiles to their post-generation state with no formatting applied